firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3725,6 +3725,20 @@ extension String {
         tableName: nil,
         value: "View",
         comment: "Label for an action used to view one or more tabs from a notification.")
+
+// MARK: - Close tab notifications
+    public static let CloseTab_ArrivingNotification_title = MZLocalizedString(
+        key: "CloseTab_ArrivingNotification_title",
+        tableName: "nil",
```

Comment:
All new strings should have a table name. Could you please add one for both strings?

Reply:
Done!